romana-install/roles/romana/kubernetes-postinstall/files/demo/demo.sh

```diff
@@ -21,7 +21,7 @@ function maybe_first_prompt() {
 }
 function run() {
     maybe_first_prompt
-    rate=25
+    rate=$((25 * (1 + ${#1} / 30)))
```

Comment:
Makes it faster when the command being "run" is really long.
It's a bit tedious watching the long commands (especially cleanup ones) go at 25bps (the rate).

So it's 25 * (1 + length of command-line / 30). Short commands run at 25bps, longer ones at 50bps or 75bps.